.github/workflows/os-zoo.yml

```diff
@@ -130,7 +132,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        branch: [openssl-3.0, openssl-3.1, master]
+        branch: [openssl-3.0, openssl-3.3, master]
```

Comment:
should we  be running these nightly on all active branches?

Reply:
I'm not sure it really brings a value.